handlers.js

```diff
@@ -99,6 +115,23 @@ let [handle_stime, handle_dtime] = both_classes(time);
 let [handle_sdem, handle_ddem] = both_classes(dem);
 let [handle_srank, handle_drank] = both_classes(rank);
 
+async function handle_online() {
+    let servers = await tempus.serverList();
+    let players = [];
+    for (let s of servers) {
+        if (!s.game_info) continue;
+        for (let p of s.game_info.players) {
+            if (!p.id || p.id == null) continue;
+            let player = await p.toPlayerStats();
```

Comment:
Would it be possible to do this in parallel via some sort of promise map or something? Seems rather annoying to have to do this all sequentially.

Reply:
Resolved in https://github.com/arispoloway/tempus-discord-bot/commit/89778b3b14169856dc46e03ee1bc5844345a5a6e